### ISSUE TITLE
Combobox - not keyboard accessible in Firefox

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#126]
As commented in https://github.com/phetsims/phet-io-wrapper-sonification/issues/82#issuecomment-505730844 this version of Molarity (https://phet-dev.colorado.edu/html/molarity/1.5.0-dev.14/phet/molarity_en_phet.html) has some issues with the combobox.

Awe, it seems that it is a Firefox issue. I am losing focus on the combobox when using Firefox (no screen reader). 
The sim seems to work fine with Safari (also without screen reader)
- When using FF on MacOS, once the list of options is open, keyboard focus is lost when trying to use the arrow keys. 

This happens on the sonified version and the non-sonified version.

**Comment by terracoda** [comment]
@jbphet, I am tagging @twant, too, as this issue is not specifically on the sonified version as I had originally thought. It is something to do with Firefox and the arrow keys.

**Comment by terracoda** [comment]
@jbphet and @twant, I also tested in Chrome (my Chrome has ChromeVox on by default) and there are no issues with the arrows and the Combobox using ChromeVox.

This indeed, seems like Firefox issue.

**Comment by jessegreenberg** [comment]
Assigning to myself since this seems like a keyboard navigation issue. I am not seeing any issues when I run the sim with Firefox in Windows 10, I will try on a macOS machine soon.

**Comment by jbphet** [comment]
I tested this on the PhET device "Jordan", MacOS version 10.13.6, Firefox version 67.0.3, and did not see the problem that @terracoda described.  I was able to navigate through the items in the combobox just fine using the arrow keys without any loss of focus.

**Comment by jbphet** [comment]
@terracoda - can we get MacOS and Firefox versions for the configuration(s) where you saw this problem?

**Comment by terracoda** [comment]
Sure @jbphet 
- Mac OS High Sierra 10.13.6
- FireFox Quantum 10.13.6

**Comment by terracoda** [comment]
@jbphet , I just updated my previous comment, apologies for any confusion.

**Comment by jessegreenberg** [comment]
I just tested macOS 10.14.5 with FF 67 with the dev version linked here and master and didn't see the reported problem.

@KatieWoe do you have access to macOS 10.13? If so, could you try to use the combobox at https://phet-dev.colorado.edu/html/molarity/1.5.0-dev.14/phet/molarity_en_phet.html with a keyboard and let us know if it works?

**Comment by KatieWoe** [comment]
It works if you use space, but not if you use enter.

**Comment by jessegreenberg** [comment]
@KatieWoe and I talked over slack, notes in https://github.com/phetsims/molarity/issues/126#issuecomment-510256980 were from testing the checkbox rather than the combobox. When testing the combobox, @KatieWoe said
> Just tried on 10.13, and space and enter would both open it just fine, using the arrow keys let you move to a selection, and space or enter would let you select.
Same with 10.14.5

@KatieWoe can you please confirm that you were testing in Firefox?

**Comment by KatieWoe** [comment]
@jessegreenberg yes I was

**Comment by terracoda** [comment]
@terracoda, I am still having the same issue (loss of focus) when moving through the list on MacOS 10.13.6 and my Firefox just updated to Quantum 68.0 64-bit.

**Comment by jessegreenberg** [comment]
OK, thanks @KatieWoe.

@terracoda since we aren't able to reproduce this issue anywhere else I think the next step will be to debug on your machine over a voice call to see if we can get any additional information about what is causing this.

**Comment by jessegreenberg** [comment]
@terracoda and I investigated this today - as it turns out, there is an a11y setting within Firefox that allows you to navigate text content with arrow keys without even using a screen reader. That is why the arrow keys were making focus go away. With this setting, the problem happens for other items in the sim too. Focus is lost after pressing arrow keys if focus was on buttons or checkboxes.

Disabling "Always use the cursor keys to navigate within pages" fixed the problem. I added documentation for this in https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/edit, where we document other such issues.
